receiver/hostmetricsreceiver/internal/scraper/diskscraper/generated_package_test.go

```diff
@@ -3,9 +3,8 @@
 package diskscraper
 
 import (
-	"testing"
-
 	"go.uber.org/goleak"
+	"testing"
```

Comment:
you can revert those changes. You can run `make generate` at the root of the checkout to do so, or just revert this file to main `git checkout main receiver/hostmetricsreceiver/internal/scraper/diskscraper/generated_package_test.go`

Reply:
This should be fixed now @atoulme